src/main/java/net/dries007/tfc/common/blocks/TFCBlocks.java

```diff
@@ -9,15 +9,18 @@
 import java.util.Map;
```

Comment:
fwiw, this class (along with `TFCItems` is organized by item group. Please verify that it still is.